roles/confluent.test/upgrade_testing/confluent-kafka-kerberos-customcerts-rhel-upgrade-550-current.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
all these files have duplicating code. There should be one file `upgrade.sh` that takes arguments... something like:

```
SCENARIO_NAME=$1
START_BRANCH=$2
END_BRANCH=$3
```

And then your other files, can call it with
```
upgrade.sh confluent-kafka-kerberos-customcerts-rhel 5.5.0-post 6.0.0-post
```

Reply:
@domenicbove How does the user know which scenarios are supported for upgrade testing with that approach?  Because we only support a specific subset, I think abstracting it this way is actually far more confusing if someone wants to run these in the future.

Reply:
Good question, lets say someone runs upgrade.sh on a scenario that does not exist, then the script should error out saying exactly that

Reply:
So we would need to modify the script to include a list of accepted scenarios and if the parameter doesn't match error out.. that's doable.

Reply:
A list might not be necessary, because lets say you run 
```
git checkout 5.5.0-post  #or any other branch
molecule converge -s some-scenario-that-does-not-exist-in-this-branch
```

then molecule itself will error out, saying can't find scenario

Reply:
Thinking about this further, we would have to setup a list of scenarios, a list of versions, then validate each across scenario version and error out if it's not a match.  Honestly, it seems like a lot of effort for no gain.

Reply:
Yeah some of the scenarios do exist across versions however may not contain all of the required zookeeper instances, for example.  The more I think about this, the more I think we are overcomplicating it to save a few lines of code.

Reply:
What happens when you run the script against a scenario with only 1 zk?

Reply:
It fails because the ZK upgrade can't run, it can't pass the health check.  That's why I needed to update the subset of scenarios to have 3 ZK hosts.

Reply:
makes sense, but that too is by design of the upgrade playbook so sorta a success in a way. I think we are getting off track though. my initial comment was just about making a helper script with arguments that your other scripts can call into so that duplicating code is removed

Reply:
I completely mis-read your first comment, apologies, I thought you wanted one script that the user would feed parameters too.  I understand what you mean now, create another script that the originals call and just pass their parameters too.